app/models/user.rb

```diff
@@ -13,13 +13,24 @@ class User < ActiveRecord::Base
          :trackable,
          :validatable
 
-  validates :username, presence: true,
+  validates :username,
+            presence: true,
             uniqueness: true,
             length: { in: 3..20 }
   validates :website, :allow_blank => true,
-            format: { with: URI.regexp },
-            if: Proc.new { |a| a.website.present? }
+            format: {
+              with: URI.regexp },
+              if: Proc.new { |a| a.website.present?
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.